aries-site/src/pages/components/textinput.js

```diff
@@ -28,7 +28,7 @@ const TextInput = () => (
       <Subsection name={title} level={1} topic={topic}>
         <SubsectionText>{page.description}</SubsectionText>
         <Example
-          designer="https://designer.grommet.io/textinput?id=HPE-design-system-taylor-seamans-hpe-com&mode=edit"
```

Comment:
should we switch out links ? for the main designer

Reply:
It should be updated

Reply:
okay good to know!